packages/init/.gitignore

```diff
@@ -1,4 +1,2 @@
-node_modules
-build
-coverage
-.caches
\ No newline at end of file
+/dist
+/playground/*
```

Comment:
what's playground?

are caches used?
are coverage checkers used?

Reply:
- Playground — answered here: https://github.com/prismicio/slice-machine/pull/799#discussion_r1119489754
- Caches are not used in ESLint if that's what you're asking. Vite and Vitest use some caches, but they live `node_modules`.
- Coverage checkers are used with Vitest + c8. Reports are ignored in the root-level `.gitignore`: https://github.com/prismicio/slice-machine/blob/e501e89ef5fd54cf4607e688e9cad4cdc7c58890/.gitignore#L37